docs/api/mcp/index.mdx

```diff
@@ -0,0 +1,273 @@
+# Sourcegraph MCP Server
+
+<p className="subtitle">Connect AI agents and applications to your Sourcegraph instance's code search and analysis capabilities.</p>
+
+The Sourcegraph Model Context Protocol (MCP) Server provides AI agents and applications with programmatic access to your Sourcegraph instance's code search, navigation, and analysis capabilities through a standardized interface.
+
+<Callout type="info">MCP Server requires Sourcegraph 6.8+ and is available on all deployment types.</Callout>
```

Comment:
I don't think we need to call out either of those.

1. The docs are versioned and clearly say 6.8 in the top right corner
2. If ALL deployment types work, why mention it?